astropy/wcs/docstrings.py

```diff
@@ -2232,6 +2232,29 @@ def RA_DEC_ORDER(indent=0):
 Sect. 5.2.1), integer (Sect. 5.2.3), and floating-point values
 (Sect. 5.2.4) for all keywords.
 
+
+.. warning::
+    There are two possible ways of writing to the attributes on this class::
```

Comment:
In addition, even if you look at this as an _acceptable way_ of _setting_ attributes (when it actually is a modification of the object returned by a getter), it would not apply to all attributes in this class. For example, "second way" would not work on `Wcsprm.name` or `Wcsprm.velosys`. Therefore, I think this note should be more specific about parameters to which it applies.